### DISCUSSION TITLE
WPF: can't run on .NET 4.8

### DISCUSSION THREAD

**Discussion by rabbitLove520** [cefsharp/CefSharp#3324] [General]
Cefsharp.WPF : `86.0.241`

my app target .NET framework is .NET 4.5.2, but it can not run on the latest Win 10 with .NET 4.8
here is the log:
```
Application: *****.exe Framework version: v4.0.30319 说明: 由于未经处理的异常，进程终止。异常信息:  
System.IO.FileNotFoundException at *******.App..ctor() at Baoxiaohe.App.Main()
--
```

**Comment by amaitland** [comment]
English only please. 

Do you Visual C++ 2015 or greater installed? Please confirm you have deployed all the required dependencies https://github.com/cefsharp/CefSharp/wiki/Output-files-description-table-%28Redistribution%29

**Reply by rabbitLove520** [reply]
@amaitland tks, you solve the problem. however, how can I build the application along with the VC++ 2015 Runtime?

**Reply by amaitland** [reply]
Start by reading https://docs.microsoft.com/en-us/cpp/windows/redistributing-visual-cpp-files?view=msvc-160

See also https://github.com/cefsharp/CefSharp/wiki/Frequently-asked-questions#6-how-do-i-include-the-visual-studio-c-201220132015-redistributables-with-my-application
